TTYC.Application/Products/AddProduct/AddProductHandler.cs

```diff
@@ -27,8 +27,8 @@ public async Task<Guid> Handle(AddProductCommand command, CancellationToken canc
                 Description = command.Description,
                 DefaultPriceData = new ProductDefaultPriceDataOptions
                 {
-                    Currency = "usd",
-                    UnitAmountDecimal = command.Price * 100
+                    Currency = PaymentOptions.Usd,
+                    UnitAmountDecimal = command.Price * PaymentOptions.Amount
```

Comment:
Amount in constants is an overkill, because it is always 100 cents in dollar and you will never need to change that coefficent, but Usd could be changed because of, e.g. changing case-sensitivity in core API